src/interactive/repl.ts

```diff
@@ -900,9 +915,11 @@ async function evaluate(editor: vscode.TextEditor, range: vscode.Range, text: st
             }
             r.setContent(results.resultContent(' ' + result.inline + ' ', result.all, Boolean(result.stackframe)))
         }
+
+        return true
     } catch (err) {
         r.remove(true)
-        telemetry.handleNewCrashReportFromException(err, 'Extension')
+        throw(err)
```

Comment:
I think this function is only being called from functions that already have the error handler set up, we should in general probably only ever send crash reports from the most outer exception handler.